test/Conversion/MooreToCore/basic.mlir

```diff
@@ -868,3 +868,15 @@ moore.module @Assert(in %cond : !moore.l1)  {
   moore.return
   }
 }
+
+// CHECK-LABEL: hw.module @StringConstant
+moore.module @StringConstant() {
+  moore.procedure initial {
+    // CHECK: hw.constant 1415934836
+    %str = moore.string_constant "Test" : i2
+    // CHECK: hw.constant 0
+    %str_empty = moore.string_constant "" : i128
```

Comment:
Maybe add the `: i2` and `: i128` type suffices to the CHECK lines, to make sure the conversion does the right thing.

Reply:
Done. Corrected types as mentioned [above](https://github.com/llvm/circt/pull/7752#discussion_r1825464079) and added type suffixes to the `CHECK` lines.